### PR TITLE
docs: add COEX Duocam russian docs

### PR DESCRIPTION
**Pull request by kpetrykin**
This is a short how-to about using Dualcam camera

### CODE REVIEW

File: docs/ru/duocam.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+QGC c необходимый функционалом для управления "COEX DuoCam" можно скачать в нашем [репозитории](https://github.com/CopterExpress/qgroundcontrol/releases). Скачивать следует тот релиз, в названии которого присутствует слово `duocam` с максимальным порядковым номером.
+
+### Как использовать
+
+При открытии виджета QGC пошлет MAVLink-сообщения с запросом текущих значений свойств. Необходимо дождаться, пока они все до конца загрузятся. Если загрузка не завершается в течение минуты, необходимо нажать кнопку "Refresh values".
+
+При изменении значений свойств QGC осуществляет коммуникацию с DuoCam посредством протокола MAVLink, поэтому нет смысла очень часто нажимать контролы - наоборот, после нажатия следует дождаться изменения картинки и только потом переходить к следующему свойству.
+
+[Подробнее об архитектуре DuoCam и программе duocam-mavlink](duocam_mavlink.md).
```
Comment (okalachev) [inline docs/ru/duocam.md:98]: Broken link.

**Review by okalachev — CHANGES_REQUESTED** [review]
`gitup_seek.png` is too large. Change the resolution or/and use JPG.

**Review by kpetrykin — COMMENTED** [review]
It looks like I need to install russian spellchecker

**Review by okalachev — CHANGES_REQUESTED** [review]
https://travis-ci.org/github/CopterExpress/clover/jobs/740962993#L935.

Also, `duocam_mavlink.md` is not added to summary.

**Comment by kpetrykin** [comment]
I don't think we need a direct link to duocam-mavlink article here because it is conceptually a part of COEX Duocam article.

As for linter, it fails on "duocam-mavlink" string which is a name of uttility and writing it as "duocam-MAVLink" would be wrong.

**Comment by okalachev** [comment]
Any article should be in the summary to be accessible. That's how gitbook works.

Symbols (like a utility binary name) should be places into ` symbols, this way they would be linted by markdown-lint. Or you can suppress this lint using `` syntax.

Documents that don't pass the tests cannot be merged.

**Pull request merged**